bindings/ios/MEGATransfer.mm

```diff
@@ -163,4 +163,8 @@ - (MEGATransferState)state {
     return (MEGATransferState) (self.megaTransfer ? self.megaTransfer->getState() : 0);
 }
 
+- (unsigned long long)priority {
+    return (MEGATransferState) (self.megaTransfer ? self.megaTransfer->getPriority() : 0);
```

Comment:
Looks like this casting to MEGATransferState is wrong.

Reply:
fixed